prog/mutation.go

```diff
@@ -45,26 +45,24 @@ func (p *Prog) Mutate(rs rand.Source, ncalls int, ct *ChoiceTable) {
 				}
 				s := analyze(ct, p, c)
 				for stop := false; !stop; stop = r.bin() {
-					args, bases, parents := mutationArgs(c)
+					args, bases, _ := mutationArgs(c)
```

Comment:
Remove parents return value from mutationArgs entirely. That function is called only here, let's not accumulate unused code.